sparkly/writer.py

```diff
@@ -255,6 +275,188 @@ def write_partition_to_kafka(messages):
 
         rdd.mapPartitions(write_partition_to_kafka).count()
 
+    def redis(self,
```

Comment:
Reader would not be hard to add, but doesn't seem like a generic usecase. On the other hand, it seems much more natural to write from Spark to Redis. I don't really understand what "very specific" means in this context.. I think it's as specific as exporting to Cassandra or ES.